templates/index.html

```diff
@@ -1,3 +1,4 @@
+{# Templates should inherit from base rather than duplicating code#}
```

Comment:
@shriyanka Holy Foosh! Sorry - did not check the other templates. I am used to seeing `base.html` from which all other template extends. 😝